docs/tutorial/fdfitting.rst

```diff
@@ -303,10 +311,12 @@ Calculating per point contour length
 Sometimes, one wishes to invert the model with respect to one parameter (i.e. re-estimate one parameter on a per data
 point basis). This can be used to obtain dynamic contour lengths::
 
-    file3 = lk.File("20200430-182304 FD Curve 40.h5")
+    file3 = lk.File("fd_multiple_lc.h5")
```

Comment:
No strong preference from my side. I'll change it. Just remember that when the zip file goes up, it needs to be correct since linux/mac are case sensitive.